api/api.go

```diff
@@ -207,6 +207,7 @@ func New(requiredUserAgent string, requiredPassword string, cs modules.Consensus
 		router.GET("/renter", api.renterHandlerGET)
 		router.POST("/renter", RequirePassword(api.renterHandlerPOST, requiredPassword))
 		router.GET("/renter/contracts", api.renterContractsHandler)
+		router.GET("/renter/contract/*contractid", api.renterContractDetailHandler)
```

Comment:
If we are going to be feeding historical contracts out of this endpoint it should probably take a parameter to specify if historical should be included or not, as the historical contracts could be a lot more data - thousands even depending on how your renter is configured